sdk/src/time_stamp.rs

```diff
@@ -660,6 +691,54 @@ pub(crate) fn get_timestamp_response(tsresp: &[u8]) -> Result<TimeStampResponse>
     Ok(ts)
 }
 
+fn get_tst_info_from_signed_data(signed_data: &SignedData) -> Result<Option<TstInfo>> {
+    if signed_data.content_info.content_type == OID_CONTENT_TYPE_TST_INFO {
+        if let Some(content) = &signed_data.content_info.content {
+            Ok(Some(
+                Constructed::decode(content.to_bytes(), bcder::Mode::Der, |cons| {
+                    TstInfo::take_from(cons)
+                })
+                .map_err(|_err| Error::CoseTimeStampGeneration)?,
+            ))
+        } else {
+            Ok(None)
+        }
+    } else {
+        Ok(None)
+    }
+}
+
+// Get TimeStampResponse from DER TimeStampResp bytes or timeStampToken
+pub(crate) fn get_timestamp_signed_data(data: &[u8]) -> Result<Option<SignedData>> {
```

Comment:
FYI once PR #696 lands, this function will be known as `c2pa_crypto::time_stamp::signed_data_from_time_stamp_response`.

Reply:
Nvm. That wasn't quite accurate, but I fixed that up when I did the merge.